boa/src/builtins/object/internal_methods.rs

```diff
@@ -352,7 +352,7 @@ impl Object {
 
     /// Helper function for property insertion.
     #[inline]
-    pub(crate) fn insert_property<N>(&mut self, name: N, p: Property)
+    pub(crate) fn insert<N>(&mut self, name: N, p: Property)
```

Comment:
I think we should make `.insert` to be a private function to `Object`.
```suggestion
    fn insert<N>(&mut self, name: N, p: Property)
```